onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -164,6 +164,7 @@ static constexpr PATH_TYPE MODEL_URI = TSTR("testdata/mul_1.onnx");
 static constexpr PATH_TYPE SEQUENCE_MODEL_URI = TSTR("testdata/sequence_length.onnx");
 static constexpr PATH_TYPE CUSTOM_OP_MODEL_URI = TSTR("testdata/foo_1.onnx");
 static constexpr PATH_TYPE CUSTOM_OP_LIBRARY_TEST_MODEL_URI = TSTR("testdata/custom_op_library/custom_op_test.onnx");
+static constexpr PATH_TYPE CUSTOM_CUDA_OP_LIBRARY_TEST_MODEL_URI = TSTR("testdata/custom_op_library/custom_cuda_op_test.onnx");
```

Comment:
nit: we may totally avoid adding a new test model in the repo by just using the existing custom op below with the newly added ifdef.